lib/slack/setup.js

```diff
@@ -20,7 +20,7 @@ module.exports = function slackSetup(api, bot, logError, optionalParser, optiona
       ||
       (request.post.trigger_word && request.post.token === request.env.slackWebhookToken))
       return bot(parser(request.post), request)
-        .then(responder)
+        .then((botResponse) => responder(botResponse, request.env.slackToken))
```

Comment:
Why do we need to pass `request.env.slackToken` to responder? As far as I can see [here](https://github.com/claudiajs/claudia-bot-builder/blob/master/lib/slack/reply.js#L7) this doesn't do anything and it also has side effects.